src/Homework7/Task3.java

```diff
@@ -0,0 +1,38 @@
+package Homework7;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Task3 {
+    public static void main(String[] args) {
+        Set<Integer> set1 = new HashSet<>();
```

Comment:
It's the right way to initialize a set. But you can also do it in one line like this `Set<Integer> set1 = new HashSet<>(Arrays.asList(1, 50, 3));`.

Reply:
Thanks for this tip, looks much better.